pacmacs-score.el

```diff
@@ -50,21 +50,23 @@
     (let ((inhibit-read-only t))
       (erase-buffer)
       (-> (pacmacs--read-score-table)
-          (pacmacs--sort-score-table)
           (pacmacs--render-score-table)))))
 
 (defun pacmacs--read-score-table ()
   (when (file-exists-p pacmacs--score-file-name)
-    (-> pacmacs--score-file-name
-        (f-read-text)
-        (read-from-string)
-        (car))))
+    (->> pacmacs--score-file-name
+         (f-read-text)
+         (read-from-string)
+         (car)
+         (pacmacs--sort-score-table))))
```

Comment:
Why sort table after reading it? Maybe you don't need it because it will always be written in the sorted state?